commit 45b2c57d345092e8a6a9f065a44c9801e09c24c5
Author: Siddhesh Poyarekar <siddhesh@sourceware.org>
Date:   Mon Mar 15 16:00:06 2021 +0530

    support: Typo and formatting fixes
    
    - Add a newline to the end of error messages in transfer().
    - Fixed the name of support_subprocess_init().
    
    (cherry picked from commit 95c68080a3ded882789b1629f872c3ad531efda0)

diff --git a/support/support_capture_subprocess.c b/support/support_capture_subprocess.c
index a7afa0e70b48f649..3eb825b9afc0b5e6 100644
--- a/support/support_capture_subprocess.c
+++ b/support/support_capture_subprocess.c
@@ -36,7 +36,7 @@ transfer (const char *what, struct pollfd *pfd, struct xmemstream *stream)
       if (ret < 0)
         {
           support_record_failure ();
-          printf ("error: reading from subprocess %s: %m", what);
+          printf ("error: reading from subprocess %s: %m\n", what);
           pfd->events = 0;
           pfd->revents = 0;
         }
diff --git a/support/support_subprocess.c b/support/support_subprocess.c
index 88489a3357f6aae0..838eda96ffd6eb9a 100644
--- a/support/support_subprocess.c
+++ b/support/support_subprocess.c
@@ -27,7 +27,7 @@
 #include <support/subprocess.h>
 
 static struct support_subprocess
-support_suprocess_init (void)
+support_subprocess_init (void)
 {
   struct support_subprocess result;
 
@@ -48,7 +48,7 @@ support_suprocess_init (void)
 struct support_subprocess
 support_subprocess (void (*callback) (void *), void *closure)
 {
-  struct support_subprocess result = support_suprocess_init ();
+  struct support_subprocess result = support_subprocess_init ();
 
   result.pid = xfork ();
   if (result.pid == 0)
@@ -71,7 +71,7 @@ support_subprocess (void (*callback) (void *), void *closure)
 struct support_subprocess
 support_subprogram (const char *file, char *const argv[])
 {
-  struct support_subprocess result = support_suprocess_init ();
+  struct support_subprocess result = support_subprocess_init ();
 
   posix_spawn_file_actions_t fa;
   /* posix_spawn_file_actions_init does not fail.  */
